css/main.css

```diff
@@ -56,19 +61,177 @@ textarea {
    Author's custom styles
    ========================================================================== */
 
+ul {
+  list-style-type: none;
+  padding: 0;
+}
+
+.black {
+  color: #000;
+}
+
+body {
+  width: 100%;
+  height: 100%;
+}
+
+.main {
+  display: flex;
+  overflow: hidden;
+  font: normal 16px sans-serif;
+  width: 100%;
+  height: 100%;
+}
+
+.sidebar {
+  display: flex;
+  flex-direction: column;
+  background: #0e171d;
+  color: #fff;
+  width: 300px;
+  height: 100%;
+  float: left;
+}
+
+.logo-motorpool {
+  background-image: url("../img/logo.png");
+  background-repeat: no-repeat;
+  margin-left: 35px;
+  margin-top: 40px;
+  margin-bottom: 46px;
+  min-width: 67px;
+  min-height: 43px;
+}
+
+form.search-form {
+  padding-left: 26px;
+  margin-bottom: 14px;
+}
+
+form.search-form input {
+  background: #0e171d;
+  color: #fff;
+  border: 1px solid #1e2e3a;
+  border-radius: 20px;
+  padding: 12px 18px;
+  outline: none;
+}
+
+form.search-form input:focus {
+  background: #0c1218;
+}
+
+form.search-form input:placeholder {
+  color: #ff0000;
+}
+
+form.search-form button[type="submit"] {
+  background-image: url("../img/search-icon.png");
+  background-repeat: no-repeat;
+  border: none;
```

Comment:
Sometime FEDs start a project with something like http://meyerweb.com/eric/tools/css/reset/ which automatically removes pesky button styles, UL bullets, etc before you begin your work.